README.md

```diff
@@ -55,16 +55,20 @@ Here are some highlights:
 - [AirWatch](https://github.com/jupiterone/graph-airwatch)
 - [AquaSec](https://github.com/jupiterone/graph-aquasec)
 - [JFrog Artifactory](https://github.com/jupiterone/graph-artifactory)
+- [atSpoke](https://github.com/jupiterone/graph-atspoke)  
 - [Auth0](https://github.com/jupiterone/graph-auth0)
 - [Azure](https://github.com/jupiterone/graph-azure)
 - [Azure DevOps](https://github.com/jupiterone/graph-azure-devops)
 - [BambooHR](https://github.com/jupiterone/graph-bamboohr)
 - [Bugcrowd](https://github.com/jupiterone/graph-bugcrowd)
 - [CbDefense](https://github.com/jupiterone/graph-cbdefense)
+- [Checkmarx](https://github.com/jupiterone/graph-checkmarx)
 - [Cisco Amp](https://github.com/jupiterone/graph-cisco-amp)
 - [Cisco Meraki](https://github.com/jupiterone/graph-cisco-meraki)
 - [Cloudflare](https://github.com/jupiterone/graph-cloudflare)
+- [Cobalt](https://github.com/jupiterone/graph-cobalt)
 - [CrowdStrike](https://github.com/jupiterone/graph-crowdstrike)
+- [CyberReason](https://github.com/jupiterone/graph-cyberreason)
```

Comment:
Let's remove this one. It actually isn't a completed integration.

Reply:
Removed in latest commit.